src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -17,7 +17,7 @@ public class AuthenticationTests
         private const string TestUser = "testuser";
         private const string TestPassword = "testpassword";
 
-        [Theory]
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.HasHttpApi))]
```

Comment:
@Priya91, @JohnChen0, I still see this HasHttpApi checking IsWindows... doesn't that mean that even though the PlatformSpecific was removed from the test class, none of these tests are actually running on Unix now?